### PR TITLE
[KOGITO-2167] - Add soft link of docker pointing to buildah for authe…

### PR DESCRIPTION
**Pull request by Kaitou786**
…nticating with quay.io registry using `withDockerRegistry` plugin in jenkins.

See: https://issues.redhat.com/browse/KOGITO-2167

Signed-off-by: TARUN KHANDELWAL <tarkhand@redhat.com>

### CODE REVIEW

**Comment by radtriste** [comment]
What is the purpose of this ?

**Review by radtriste — APPROVED** [review]

**Review by msmagnanijr — APPROVED** [review]

**Comment by Kaitou786** [comment]
@spolti can you please take a look as well?

File: kogito-cloud-operator-jenkins-slave/image.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: kogito-cloud-operator-jenkins-slave
+name: quay.io/redhat/kogito-cloud-operator-jenkins-slave
```
Comment (spolti) [inline kogito-cloud-operator-jenkins-slave/image.yaml]: I think we should keep it that way, with no quay.io before, IIRC we'll not be able to use the redhat org anymore.

**Review by spolti — APPROVED** [review]

**Review by spolti — CHANGES_REQUESTED** [review]
oops, approved by mistake :)

File: kogito-cloud-operator-jenkins-slave/image.yaml (hunk 1 above)
Comment (Kaitou786) [reply kogito-cloud-operator-jenkins-slave/image.yaml]: Changed it back to as it was

**Review by spolti — APPROVED** [review]

**Pull request merged**